packages/nuxt/src/app/components/client-only.ts

```diff
@@ -41,7 +41,7 @@ export function createClientOnly<T extends ComponentOptions> (component: T) {
           : h(res)
       } else {
         const fragment = getFragmentHTML(ctx._.vnode.el ?? null)
```

Comment:
should we instead make `getFragmentHTML` return null so we can add `|| ['<div></div>']` to this line, so `fragment.length` below is `1`? (I haven't tested this so you may have a reason for doing it as below.)

Reply:
Sure ! 😅 this was just  the quickest solution i found. But modifying `getFramentHTML` to return `string[]|null` would also work !